### ISSUE TITLE
Unexpected token errors for 5.6 "constant expression" initializers

### ISSUE THREAD

**Issue by rwos** [pdepend/pdepend#238]
Initializing with arrays seems fixed since https://github.com/pdepend/pdepend/issues/192, but other valid expressions still error out.

This is with pdepend d134d20590b316dc4ed0c7958115b377c251d6bd (current HEAD):

```
$ cat test.php
<?php
class FOO {
    // none of these work
    const BAR = 1+2-3*4/5;
    const BAZ = 'hello ' . 'world';
    public $bar1 = 1+2-3*4/5 + self::BAR;
    public $bar2 = 'hello ' . 'world';
}

$ vendor/bin/phpmd --summary-xml=log test.php

[...]
The following error occured:
Unexpected token: *, line: 4, col: 22, file: /home/[...]
```

(I'm actually using this via phpmd, but I thought is probably the better place for the issue)

**Comment by zuozp8** [comment]
:+1: this blocks me from using phpmd in my project

**Comment by jschirrmacher** [comment]
+1

**Comment by tiger-seo** [comment]
will this get included into 2.2.3 ?

**Comment by manuelpichler** [comment]
@tiger-seo it's fixed in 2.2.3 which I have released yesterday evening.
